### PR TITLE
Feature/komarch 14 tags

### PR DESCRIPTION
**Pull request by yanchith**
Add crud for tags and relationships between tags and posts.

### CODE REVIEW

File: app/Http/Requests/TagRequest.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            // 'name' => 'required|min:5|max:255'
```
Comment (igor-kamil) [inline app/Http/Requests/TagRequest.php:29]: I am up for defining some rules. At least name is realy required
```suggestion
             'name' => 'required|min:2|max:255'
```

File: app/Models/Post.php [hunk 2]
```diff
@@ -22,8 +22,6 @@ class Post extends Model
         HasTranslations,
         CrudTrait;
 
-    public $with = ['tags'];
```
Comment (igor-kamil) [inline app/Models/Post.php:25]: just asking - why did you remove it? this makes eager loading. saving amount of db queries.

File: resources/views/vendor/backpack/base/inc/sidebar_content.blade.php [hunk 3]
```diff
@@ -2,6 +2,7 @@
 <li class="nav-item"><a class="nav-link" href="{{ backpack_url('dashboard') }}"><i class="la la-home nav-icon"></i> {{ trans('backpack::base.dashboard') }}</a></li>
 <li class='nav-item'><a class='nav-link' href='{{ backpack_url('post') }}'><i class='nav-icon la la-newspaper-o'></i> Posts</a></li>
 <li class='nav-item'><a class='nav-link' href='{{ backpack_url('user') }}'><i class='nav-icon la la-users'></i> Users</a></li>
+<li class='nav-item'><a class='nav-link' href='{{ backpack_url('tag') }}'><i class='nav-icon la la-question'></i> Tags</a></li>
```
Comment (igor-kamil) [inline resources/views/vendor/backpack/base/inc/sidebar_content.blade.php:5]: what about to choose nicer icon instead of `la-question` ? ;)

**Review by igor-kamil — CHANGES_REQUESTED** [review]
just a very tiny hints

File: app/Models/Post.php (hunk 2 above)
Comment (yanchith) [reply app/Models/Post.php:25]: Probably just my misunderstanding of things! I thought that when you define it with a method, it does the same, but now I looked at the docs and see this is something else. I'll put it back.

**Comment by yanchith** [comment]
@igor-kamil All done, I believe 🙂

**Review by igor-kamil — APPROVED** [review]
🌟

**Pull request merged**